Content.Server/Medical/HealingSystem.cs

```diff
@@ -126,6 +107,24 @@ entity.Comp.DamageContainerID is not null &&
             QueueDel(args.Used.Value);
         }
 
+        // start-backmen: surgery
+        // This is still pretty shitcodey, but a lot better than previous iteration.
+        // We are just trying to heal the most damaged body part.
+        if (healed != null)
```

Comment:
```suggestion
        if (healed != null && healed.GetTotal() == 0)
```
We sadly do need to check if healed.GetTotal() is 0, because the healed value is not null if their damage changes, or if theres literally no change, so this would effectively double-dip heal the person with only one topical application